src/views/Component.tsx

```diff
@@ -0,0 +1,173 @@
+import { Component, toNative, Vue } from 'vue-facing-decorator';
+import { observer } from 'mobx-vue-helper';
+
+import Button from 'primevue/button';
+import DataTable from 'primevue/datatable';
+import Column from 'primevue/column';
+import Panel from 'primevue/panel';
+import Stepper from 'primevue/stepper';
+import StepperPanel from 'primevue/steppanel';
+import Tree from 'primevue/tree';
+import DatePicker from 'primevue/datepicker';
+import FloatLabel from 'primevue/floatlabel';
+import InputText from 'primevue/inputtext';
+import Dialog from 'primevue/dialog';
+import { useConfirm } from 'primevue/useconfirm';
+import ConfirmDialog from 'primevue/confirmdialog';
+
+import StockNumber from '../components/StockNumber';
+import Image from '../components/Image';
+import ImageUploader from '../components/ImageUploader';
+import Downloader from '../components/Downloader';
+
+import { downloader } from '../models/service';
+import styles from './Component.module.less';
+
+const tree = [
+  {
+    key: '0',
+    label: 'node 0',
+    children: [
+      { key: '1.1', label: 'node 1.1' },
+      {
+        key: '1.2',
+        label: 'node 1.2',
+        children: [{ key: '2.1', label: 'node 2.1' }]
+      }
+    ]
+  }
+];
+
+@Component
+@observer
+class ComponentPage extends Vue {
+  date: any = undefined;
+  openDialog = false;
+  confirm = useConfirm();
+
+  download() {
+    const task = downloader.createTask(
+      'https://ows.blob.core.chinacloudapi.cn/$web/file/001%E6%B1%9F%E6%B3%A2.png',
+      'test'
+    );
+    task.start({ chunkSize: 1024 ** 2 / 2 });
+  }
+
+  closeDialog(data: any) {
+    this.openDialog = false;
+    console.log(data);
+  }
+
+  openConfirm() {
+    this.confirm.require({
+      header: 'Confirm',
+      message: 'Yes or No ?',
+      accept: console.info,
+      reject: console.error
+    });
+  }
```

Comment:
```suggestion
  closeDialog = (data: any) => {
    this.openDialog = false;
    console.log(data);
  };

  openConfirm = () =>
    this.confirm.require({
      header: 'Confirm',
      message: 'Yes or No ?',
      accept: console.info,
      reject: console.error
    });
```
所有事件回调都用箭头函数绑定 `this`